c_tests/unit.c

```diff
@@ -17,6 +17,7 @@
  */
 
 #include <stdlib.h>
+#include <stdbool.h>
```

Comment:
does this build on all platforms we support?

Reply:
Address in https://github.com/aws/dcv-color-primitives/pull/58